rust/platform/platform.bzl

```diff
@@ -19,6 +19,9 @@ _SUPPORTED_CPU_ARCH = [
     "x86_64",
     "riscv32",
     "riscv64",
+    "thumbv6m",
```

Comment:
Are there actually platform definitions for this in https://github.com/bazelbuild/platforms/blob/main/cpu/BUILD ?

I feel like this needs to be mapped to something or a PR to that repo needs to be added, right?

Reply:
Actually good point. I don't think these are really needed for this change anyways.